apps/astarte_data_updater_plant/config/test.exs

```diff
@@ -9,4 +9,5 @@ config :logger, :console,
   format: {PrettyLog.UserFriendlyFormatter, :format},
   metadata: [:realm, :device_id, :function]
 
+config :astarte_data_updater_plant, :astarte_instance_id, "test"
```

Comment:
Not strictly needed, but no problem imo to keep this if it's consistent (i.e. all other services have this `"test"` astarte instance id in test.exs)